### ISSUE TITLE
UnrolledRNN_Model

### ISSUE THREAD

**Issue by madanvikram** [sookinoby/generative-models#1]
(1) The encoder / embedding bits need a bit more explaining – it is fine if it is high-level.
(2) The following sentence is a bit confusing: “This neural network has 3 embedding layers (emb) for each character, followed by 3 dense layers.”
(3) The insight you are attempting to deliver with following is not clear: “In addition to the many-to-one RNN, there are other types of RNN that process such memory-based applications, including the popular sequence-to-sequence RNN: ![Alt text](images/loss.png?raw=true"Sequence to Sequence model")”
(4) “Here N inputs (3 characters) are mapped onto N outputs.” – Which code block are you referring to? I though we are predicting the 4th character – so 3 inputs are mapped to 1 output. Am I missing something here?
(5) The following sentence is not clear: “Basically, backpropagation multiplies the gradients along the computational graph in reverse direction. A detailed explanation of the problems with RNN is explainedhere.”
